templates/python-project/example_package/tests/test_team.py

```diff
@@ -0,0 +1,11 @@
+import unittest
+
+from example_package.team import Team
+
+
+class TeamTestCase(unittest.TestCase):
+    def test_monster_is_cool(self):
+        self.assertTrue(Team("monster").is_cool())
+
+    def test_jade_is_uncool(self):
+        self.assertFalse(Team("jade").is_cool())
```

Comment:
:) Hey jade is cool too.

Reply:
i can switch this to `assertTrue` and mark the test to be skipped, but i think tracking down this bug will take a while